src/routes/users.route.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const userController = require('../controllers/users.controller');
+
+const usersRouter = express.Router();
+
+usersRouter.get('/', userController.getAllUsers);
```

Comment:
Middlewares should be placed here 
```suggestion
usersRouter.get('/', yourMiddlewareToCheckAndSanitizeTheRequest, userController.getAllUsers);
```